core/primitives-core/src/version.rs

```diff
@@ -262,10 +262,6 @@ impl ProtocolFeature {
             ProtocolFeature::StateSyncHashUpdate => 144,
             ProtocolFeature::SimpleNightshadeV4 => 145,
             ProtocolFeature::BandwidthScheduler => 146,
-
-            // Features that are not yet in Nightly.
```

Comment:
I guess this comment is intended to be permanent so people put pre-nightly features below